[BUG]テストエラー

## バグの説明
CI環境で以下のエラーが出ている。

```
tests/test_ibetstraightbond.py::test_deploy_normal_1 FAILED                                                                                                                                              [  1%]
>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>> traceback >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>

web3 = <web3.main.Web3 object at 0x10a072fd0>, chain = <populus.chain.external.ExternalChain object at 0x109c2de10>
users = {'admin': '0xc194a6A7EeCA0A57706993e4e4Ef4Cf1a3434e51', 'agent': '0x955e1af5bf276e262386a5E9fd9b4356cBe77153', 'issuer': '0xf2AAce6dDCa4161ccc061C1e0562a8CAF2FB1867', 'trader': '0x28CFC138B7fF3AFEC69b827E3dFFADeE59cE0b68'}
bond_exchange = <web3.contract.PopulusContract object at 0x10a0dde80>

    def test_deploy_normal_1(web3, chain, users, bond_exchange):
        account_address = users['issuer']
        deploy_args = init_args(bond_exchange.address)

        web3.eth.defaultAccount = account_address
        bond_contract, _ = chain.provider.get_or_deploy_contract(
            'IbetStraightBond',
>           deploy_args=deploy_args
        )

tests/test_ibetstraightbond.py:46:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
../../.pyenv/versions/3.6.4/envs/tmr-sc/lib/python3.6/site-packages/populus/contracts/provider.py:143: in get_or_deploy_contract
    deploy_kwargs=deploy_kwargs,
../../.pyenv/versions/3.6.4/envs/tmr-sc/lib/python3.6/site-packages/populus/contracts/provider.py:127: in deploy_contract
    return self.get_contract(contract_identifier), deploy_transaction_hash
../../.pyenv/versions/3.6.4/envs/tmr-sc/lib/python3.6/site-packages/populus/contracts/provider.py:92: in get_contract
    contract_addresses,
../../.pyenv/versions/3.6.4/envs/tmr-sc/lib/python3.6/site-packages/eth_utils/functional.py:22: in inner
    return callback(fn(*args, **kwargs))
../../.pyenv/versions/3.6.4/envs/tmr-sc/lib/python3.6/site-packages/populus/contracts/provider.py:40: in filter_addresses_by_bytecode_match
    verify_contract_bytecode(web3, expected_bytecode, address)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

web3 = <web3.main.Web3 object at 0x10a072fd0>, expected_bytecode = '0x', address = '0x4675e1c6100a5caBF54F19cA1c04b9d63703ae9E'

    def verify_contract_bytecode(web3, expected_bytecode, address):
        """
        TODO: write tests for this.
        """
        from populus.contracts.exceptions import BytecodeMismatch

        # Check that the contract has bytecode
        if expected_bytecode in EMPTY_BYTECODE_VALUES:
            raise ValueError(
>               "Contract instances which contain an address cannot have empty "
                "runtime bytecode"
            )
E           ValueError: Contract instances which contain an address cannot have empty runtime bytecode

../../.pyenv/versions/3.6.4/envs/tmr-sc/lib/python3.6/site-packages/populus/utils/contracts.py:180: ValueError
>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>> entering PDB >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
> /Users/yoshihitoaso/.pyenv/versions/3.6.4/envs/tmr-sc/lib/python3.6/site-packages/populus/utils/contracts.py(180)verify_contract_bytecode()
-> "Contract instances which contain an address cannot have empty "
```

## 再現方法
ローカルのpytestでも発生する。

## 正しい挙動（想定）
テストがすべて通る状態。

--- a/contracts/IbetStraightBond.sol
+++ b/contracts/IbetStraightBond.sol
@@ -18,6 +18,7 @@ contract IbetStraightBond is Ownable, IbetStandardTokenInterface {
     string public returnAmount; // リターン内容
     string public purpose; // 発行目的
     string public memo; //メモ欄
+    bool public status; // 取扱ステータス(True：有効、False：無効)
 
     // 償還状況
     bool public isRedeemed;
@@ -45,6 +46,9 @@ contract IbetStraightBond is Ownable, IbetStandardTokenInterface {
 
     // イベント：償還
     event Redeem();
+
+    // イベント：ステータス変更
+    event ChangeStatus(bool indexed status);
 
     // コンストラクタ
     constructor(string memory _name, string memory _symbol,
@@ -221,9 +225,16 @@ contract IbetStraightBond is Ownable, IbetStandardTokenInterface {
         return image_urls[_class];
     }
 
-    // メモ欄を更新する
+    // ファンクション：メモ欄を更新する
     function updateMemo(string memory _memo) public onlyOwner() {
         memo = _memo;
+    }
+
+    // ファンクション：ステータスの有効・無効を更新する
+    // オーナーのみ実行可能
+    function setStatus(bool _status) public onlyOwner() {
+        status = _status;
+        emit ChangeStatus(status);
     }
 
 }

